### ISSUE TITLE
CSI secrets driver does not exist

### ISSUE THREAD

**Issue by burtlo** [hashicorp/secrets-store-csi-driver-provider-vault#25]
Following the instructions to [install the CSI secrets driver](https://github.com/kubernetes-sigs/secrets-store-csi-driver#install-the-secrets-store-csi-driver) creates a CSIDriver named `secrets-store.csi.k8s.io` and not `secrets-store.csi.k8s.com`.

The CSI Driver mentioned throughout a lot of the examples is `secrets-store.csi.k8s.com`.

Like the pod that is defined in the README.

```
kind: Pod
apiVersion: v1
metadata:
  name: nginx-secrets-store-inline
spec:
  containers:
  - image: nginx
    name: nginx
    volumeMounts:
    - name: secrets-store-inline
      mountPath: "/mnt/secrets-store"
      readOnly: true
  volumes:
    - name: secrets-store-inline
      csi:
        driver: secrets-store.csi.k8s.com
        readOnly: true
        volumeAttributes:
          secretProviderClass: "vault-foo"
```

**Comment by burtlo** [comment]
https://github.com/kubernetes-sigs/secrets-store-csi-driver/issues/170#issuecomment-598850519

**Comment by ritazh** [comment]
Closed via #26
